lib/sqlitex.ex

```diff
@@ -19,12 +19,7 @@ defmodule Sqlitex do
     :esqlite3.exec(sql, db)
   end
 
-  def query(db, sql, opts \\ []) do
-    case :esqlite3.prepare(sql, db) do
-      {:ok, statement} -> bind_and_query(statement, opts)
-      {:error, _}=error -> error
-    end
-  end
+  def query(db, sql, opts \\ []), do: Sqlitex.Query.query(db, sql, opts)
```

Comment:
I just learned about [defdelegate](http://elixir-lang.org/docs/stable/elixir/Kernel.html#defdelegate/2) yesterday:

```elixir
defdelegate query(db, sql, opts \\ []), to: Sqlitex.Query
```

Doesn't make much of a difference here, but it is great when you are wrapping a LOT of functions with many arguments.